src/controllers/GiveCommandHandler.ts

```diff
@@ -0,0 +1,63 @@
+export default class GiveCommandHandler {
+    points: string
```

Comment:
consider
change points=> value or vice versa
receiverUser => receiver (cause it;s already user)
giverUser=> sender or vice versa